crates/bevy_winit/src/custom_cursor.rs

```diff
@@ -17,8 +17,16 @@ pub struct CustomCursorImage {
     /// An optional texture atlas used to render the image.
     pub texture_atlas: Option<TextureAtlas>,
     /// Whether the image should be flipped along its x-axis.
+    ///
+    /// If true, the cursor's hotspot will be automatically flipped along with
+    /// the image. You don't need to adjust the `hotspot` field to account for
+    /// the flip.
     pub flip_x: bool,
     /// Whether the image should be flipped along its y-axis.
+    ///
+    /// If true, the cursor's hotspot will be automatically flipped along with
+    /// the image. You don't need to adjust the `hotspot` field to account for
+    /// the flip.
```

Comment:
```suggestion
    ///
    /// The `hotspot` will flip alongside the flipped image
```

I feel like the description has too many unnessecary words

Reply:
Missing punctuation